tests/UI/pages/commonPage.js

```diff
@@ -465,6 +465,16 @@ class CommonPage {
 
     return download.path();
   }
+
+  /**
+   * Calculate percentage
+   * @param num {number} Number to do the percentage
+   * @param per {number} Percentage value
+   * @returns {Promise<number>}
+   */
+  async percentage(num, per) {
```

Comment:
```suggestion
  percentage(num, per) {
```

Is async really needed?

Reply:
I don't really know if its needed, But Even yes, I don'tthink commonPage is the right place (it has nothing to do with pages)

Reply:
Yes async is not needed, I will delete the async.
@boubkerbribri , what's the right place ? wdyt ?